.github/workflows/ci.yml

```diff
@@ -27,7 +27,15 @@ jobs:
       # Installed *after* nightly so it is the default.
       - uses: dtolnay/rust-toolchain@stable
```

Comment:
I don't remember if this is still needed here. maybe removing it would solve the problem?